### ISSUE TITLE
runtime org.kde.Platform branch 5.15-21.08 is end-of-life

### ISSUE THREAD

**Issue by rabin-io** [Murmele/Gittyup#644]
Can we bump the run time version to the latest stable one?

```
Info: runtime org.kde.Platform branch 5.15-21.08 is end-of-life, with reason:                                                                                                                                       
   We strongly recommend moving to the latest stable version of the Platform and SDK                                                                                                                                
Info: applications using this runtime:                                                                                                                                                                              
   com.github.Murmele.Gittyup
```

**Comment by slackerbob** [comment]
I'm having the same warning message via "flatpak update" on Arch:

`
Info: runtime org.kde.Platform branch 5.15-21.08 is end-of-life, with reason:
   We strongly recommend moving to the latest stable version of the Platform and SDK
Info: applications using this runtime:
   com.github.Murmele.Gittyup

Info: runtime org.freedesktop.Platform.GL.default branch 21.08 is end-of-life, with reason:
   org.freedesktop.Platform 21.08 is no longer receiving fixes and security updates. Please update to a supported runtime version.
Info: applications using this extension:
   com.github.Murmele.Gittyup
`

**Comment by Murmele** [comment]
I am currently working on Qt6 support and then we are up to date again

https://github.com/Murmele/Gittyup/pull/429

Problem is that I don't have a windows / mac machine to fix the error

**Comment by 0verEngineer** [comment]
@Murmele Is this still the case / does this still need fixing on Mac and Windows?

**Comment by Murmele** [comment]
Yes, windows is working but the theme looks wrong and on macos it does not startup properly.

**Comment by 0verEngineer** [comment]
@Murmele On what Branch is this?

**Comment by Murmele** [comment]
@0verEngineer you can find it here:
https://github.com/Murmele/Gittyup/pull/429
